sql/core/src/test/scala/org/apache/spark/sql/streaming/MemorySinkSuite.scala

```diff
@@ -59,7 +59,7 @@ class MemorySinkSuite extends StreamTest with SharedSQLContext {
   }
 
   test("error if attempting to resume specific checkpoint") {
-    val location = Utils.createTempDir("steaming.checkpoint").getCanonicalPath
+    val location = Utils.createTempDir(namePrefix = "steaming.checkpoint").getCanonicalPath
```

Comment:
> Why this change?

Avoid to create `steaming.checkpoint` in the sql folder. I have to clean my repo after running this test.